src/Opserver.Core/Data/Exceptions/ExceptionStore.cs

```diff
@@ -185,8 +216,19 @@ public override int GetHashCode()
                 hashCode = (hashCode * -1521134295) + EqualityComparer<DateTime?>.Default.GetHashCode(EndDate);
                 hashCode = (hashCode * -1521134295) + EqualityComparer<Guid?>.Default.GetHashCode(StartAt);
                 hashCode = (hashCode * -1521134295) + EqualityComparer<Guid?>.Default.GetHashCode(Id);
+                hashCode = (hashCode * -1521134295) + EqualityComparer<string>.Default.GetHashCode(Url);
+                hashCode = (hashCode * -1521134295) + EqualityComparer<string>.Default.GetHashCode(Host);
                 return (hashCode * -1521134295) + Sort.GetHashCode();
             }
+
+            private string NormalizeDateTime(DateTime? dateTime)
```

Comment:
I tested my code again and found that I was not handling the dates properly (I thought I tested this!) on all places and the dates filters were not applying in some cases. I was missing sending a normalized value of dates on the ajax call for loadmore feature and for some of the links the exceptions page has. I've created a normalized property for dates so the different html templates that need to write the dates on the html can reuse it.

Reply:
:+1: Given the rarity this will happen, agree with the cleanliness of the approach!